common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
```

Comment:
@sarina @bradenmacdonald From a sighted keyboard only users perspective, it would be impossible to access the "clarification."  If we add ```tabindex="0"``` to this span and trigger the visual tool-tip to display on focus as well, this will work great for keyboard only users (physical or virtual) as well (like Stephen Hawking, for instance, who navigates web pages with a head switch).